index.js

```diff
@@ -125,7 +126,18 @@ var publicAgent = {
     if (!config.enabled) {
       return this;
     }
-    var logger = common.logger.create(config.logLevel, '@google/cloud-trace');
+
+    var logLevel = config.logLevel || 2;
```

Comment:
I think `initConfig` and config.js guarantee that config.logLevel will be defined by this point. At any rate, we shouldn't duplicate the default logLevel that is also in the config.

Aside: should the default logLevel in config.js continue to be an integer, or should it be `'error'`.